JULIERAJ/KIDS-FIRST-Front
# Feature/59/updating landing page

## One Line Description
Updating landing page in accordance with Figma design

## Requirements

The landing page should be updated to match the new version of the app
[https://www.figma.com/design/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=0-1&t=RI2rL2aIH4zRafp8-0](url)

## Notes
Since Home.css file was overloaded,  2 components: feature and value have been created and  moved to separate files to Components section

## Test Steps
1. Open the application.
2. Open Figma [https://www.figma.com/design/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=0-1&t=RI2rL2aIH4zRafp8-0](url)
3. Check if the landing page and all the elements are working well and the dimensions of all the elements are in accordance with Figma design.


## Checklist
